packages/api-client/src/client/ClientAPI.ts

```diff
@@ -137,4 +148,36 @@ export class ClientAPI {
 
     await this.client.sendJSON<PreKeyBundle>(config, true);
   }
+
+  /**
+   * Claim one key package for each client of the given user
+   * @param  {string} userId The user to claim the key packages for
+   * @param {string} userDomain The domain of the user
+   * @param  {string} skipOwn Do not claim a key package for the given own client
+   * @see https://staging-nginz-https.zinfra.io/api/swagger-ui/#/default/post_mls_key_packages_claim__user_domain___user_
+   */
+  public async claimMLSKeyPackages(userId: string, userDomain: string, skipOwn: string): Promise<ClaimedKeyPackages> {
+    const config: AxiosRequestConfig = {
+      data: {skip_own: skipOwn},
```

Comment:
looks like this should be included as a query, not as a request body

Reply:
nice catch man, fixed in https://github.com/wireapp/wire-web-packages/pull/4292/commits/c9819bd0fdac6430a8fbce45e57f0d10a8a14b9d

Reply:
I wonder why this endpoint is actually a `POST` endpoint :thinking: